src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.html

```diff
@@ -534,25 +534,45 @@ <h4 *ngIf="isImport" id="jhi-programming-exercise-heading-import" jhiTranslate="
                 </label>
             </div>
         </div>
-        <div class="pt-4"></div>
-        <div class="form-group" *ngIf="programmingExercise.id && !isImport">
-            <label class="form-control-label" jhiTranslate="artemisApp.exercise.notificationText" for="field_notification_text">Notification Text</label>
-            <input minlength="3" type="text" class="form-control" name="notificationText" id="field_notification_text" [(ngModel)]="notificationText" />
+        <div class="update-programming-exercise-footer align-items-center">
+            <div class="d-flex align-items-center justify-content-end" style="height: 90%; padding-right: 17%">
```

Comment:
[nit] Try to move the styling out of the html template. Either define/reuse CSS classes or use bootstrap.

Reply:
Changed in db8052e.